Cake.DocumentDb/Factories/ClientFactory.cs

```diff
@@ -26,9 +27,31 @@ public IReliableReadWriteDocumentClient GetClient()
                 {
                     ConnectionMode = ConnectionMode.Gateway,
                     ConnectionProtocol = Protocol.Https
-                }).AsReliable(new FixedInterval(
-                    15,
-                    TimeSpan.FromSeconds(200)));
+                })
+                .AsReliable(
+                    new FixedInterval(
+                        15,
+                        TimeSpan.FromSeconds(200)));
+        }
+
+        // https://github.com/Azure/azure-cosmos-dotnet-v2/blob/master/samples/documentdb-benchmark/Program.cs
+        public IDocumentClient GetClientOptimistedForWrite()
+        {
+            return new DocumentClient(
+                new Uri(settings.Endpoint),
+                settings.Key,
+                new ConnectionPolicy
```

Comment:
if this is an open-source project shouldn't we be allowing configuration rather than hardcoding these values?

Reply:
I'm assuming these values were determined from thorough experimentation (see link in comment) and am therefore not allowing them to be changed.